ipymol/core.py

```diff
@@ -52,6 +52,7 @@ def start(self):
                 server_online = True
             except:
                 pass
+        self._add_methods()
```

Comment:
What versions of PyMOL have you tried this on? Does this work on v1.86 and below?

Reply:
I checked ipymol with pymol 1.9.0 on Linux installed via conda and pymol 2.3.0 installed via homebrew.
Both cases worked. And I'm sorry that I did not check on v1.86 and below.

Thanks.